yarn.lock

```diff
@@ -2748,7 +2748,7 @@
   resolved "https://registry.yarnpkg.com/@types/ms/-/ms-0.7.31.tgz#31b7ca6407128a3d2bbc27fe2d21b345397f6197"
```

Comment:
If you run ```yarn``` (with no changes to package.json, like this PR implies), I believe these changes to the lock file should disappear.